docs/windows/SERVICE.md

```diff
@@ -0,0 +1,514 @@
+```
+
+**Check service logs:**
+```cmd
+type "C:\ProgramData\PinShare\logs\service.log"
+```
+
+**Common Issues:**
+- Port already in use (8888, 9090, 5001)
+- Antivirus blocking binaries
+- Missing write permissions to ProgramData
+
+### Debug Mode
+
+Run service in console for detailed output:
+
+```cmd
+cd "C:\Program Files\PinShare"
+pinsharesvc.exe debug
+```
+
+This shows real-time logs and errors.
+
+### Health Check Failures
+
+The service monitors IPFS and PinShare health every 30 seconds. If either fails 3 times, it stops auto-restarting.
+
+**Manual restart:**
+```cmd
+pinsharesvc.exe restart
+```
+
+**Check component status:**
+```cmd
+curl http://localhost:5001/api/v0/version     # IPFS
+curl http://localhost:9090/api/health         # PinShare
+```
+
+## Security Considerations
+
+### Firewall Rules
+
+The installer doesn't automatically configure Windows Firewall. Users may need to allow:
+
+- Port **4001** - IPFS P2P swarm
+- Port **50001** - PinShare libp2p
+
+**Add rules:**
+```powershell
+New-NetFirewallRule -DisplayName "IPFS Swarm" -Direction Inbound -Protocol TCP -LocalPort 4001 -Action Allow
+New-NetFirewallRule -DisplayName "PinShare P2P" -Direction Inbound -Protocol TCP -LocalPort 50001 -Action Allow
+```
+
+### Service Account
+
+By default, the service runs as `LocalSystem`. For enhanced security, create a dedicated service account:
+
+1. Create user: `PinShareService`
+2. Grant permissions to `C:\ProgramData\PinShare`
+3. Change service account in Services Manager
+
+### UI Access
+
+By default, the UI server binds to `localhost:8888` (localhost only). To allow remote access:
+
+1. Modify `ui_server.go` to bind to `0.0.0.0:8888`
+2. Add authentication middleware
+3. Use HTTPS (reverse proxy recommended)
+
+## Advanced Topics
+
+### Running Multiple Instances
+
+To run multiple PinShare instances on one machine:
+
+1. Install first instance normally
+2. For additional instances:
+   - Copy installation directory to different location
+   - Modify registry to use different ports
+   - Register as different service name
+   - Run installer again (custom action)
+
+### Performance Tuning
+
+**For archive nodes (many files):**
+- Set `"archive_node": true` in config.json
+- Increase IPFS repo size limit
+- Disable automatic garbage collection
+
+**For low-resource systems:**
+- Set `"enable_cache": false` in config.json
+- Reduce IPFS connection limits in IPFS config
+- Increase health check interval
+
+### Backup and Restore
+
+**Backup:**
+```powershell
+Stop-Service PinShareService
+Copy-Item "C:\ProgramData\PinShare" "D:\Backup\PinShare" -Recurse
+Start-Service PinShareService
+```
+
+**Restore:**
+```powershell
+Stop-Service PinShareService
+Remove-Item "C:\ProgramData\PinShare" -Recurse -Force
+Copy-Item "D:\Backup\PinShare" "C:\ProgramData\PinShare" -Recurse
+Start-Service PinShareService
+```
+
+## Documentation
+
+- **Installation Guide:** [`docs/windows/README.md`](docs/windows/README.md)
```

Comment:
This is a reference to *this* document. Let's either add a named achor to the specific section to which this refers or remove this line.